css-canvas-medical/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Canvas Medical</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <section id="nav-section">
+      <div class="row nav-bar">
+        <div class="column-one-fifth logo">
+          <img src="images/logo-blue.svg" alt="canvas logo" id="logo-blue" />
+        </div>
+        <div class="column-fill"></div>
+        <div class="column-one-half nav-links">
```

Comment:
This `<div class="column-fill"></div>` shouldn't be necessary since you already have `justify-content: space-between` on your `.nav-section` Another option though could be to make your `.nav-links` class have a `justify-content: flex-end` if you wanted them to be at the end of the nav container too!